app/v4/controllers/ListBsasController.scala

```diff
@@ -40,6 +40,8 @@ class ListBsasController @Inject() (val authService: EnrolmentsAuthService,
     extends AuthorisedController(cc)
     with Logging {
 
+  val endpointName = "list-bsas"
+
   implicit val endpointLogContext: EndpointLogContext =
     EndpointLogContext(controllerName = "ListBsasController", endpointName = "listBsas")
```

Comment:
could the endpointName val be used here to reduce future changes required if name changed?

Reply:
These names were like this in the main branch. I haven't made these changes, but we can discuss this with others to check why the endpoint names don't match